src/MICore/Debugger.cs

```diff
@@ -55,6 +55,24 @@ public bool IsClosed
             }
         }
 
+        public bool IsDebuggerRunning
```

Comment:
> A method instead of a property
Is this to explicitly indicate this involves some work and not just reading something off memory?